### DISCUSSION TITLE
[Question]: Does image rotation effect object detection?

### DISCUSSION THREAD

**Discussion by slashgob** [blakeblackshear/frigate#10810] [Ask A Question]
### What is your question:

I have a camera which is rotated sideways to enable me to get my drive in its full vision. I'm struggling to detect people on this camera and I'm wondering if the rotation is affecting the object detection, or if it should not matter?

People walk towards the rear of the car in the below picture but Firgate (using Coral USB TPU) struggles to identify them..

![image](https://github.com/blakeblackshear/frigate/assets/44810039/66c24ca7-83a5-4318-ac75-7cdb04424ee4)

**Comment by NickM-27** [comment]
yes, overhead views also do not work very well with the default model

**Reply by slashgob** [reply]
OK, so is it possible to have Frigate trigger an event purely on motion detection without identifying anything like a person or car?

**Reply by NickM-27** [reply]
No, events are the same as tracked objects.

We are currently rewriting the UI and there will be a dedicated page for reviewing motion detection
